integration/spark-common/src/main/scala/org/apache/carbondata/spark/rdd/CarbonRDD.scala

```diff
@@ -46,18 +46,12 @@ abstract class CarbonRDD[T: ClassTag](@transient sc: SparkContext,
       info = new CarbonSessionInfo
       info.setSessionParams(new SessionParams())
     }
+    info.getThreadParams
```

Comment:
1. Here configuration should be taken from SparkSession,sessionState
2. Remove hadoopConf from constructor